tensorflow/c/c_api.cc

```diff
@@ -1667,6 +1667,25 @@ void TF_DeleteGraph(TF_Graph* g) {
   if (del) delete g;
 }
 
+size_t TF_GraphOperationsCount(TF_Graph* graph) {
```

Comment:
I'd recommend against adding these functions. It seems that [`TF_GraphNextOperation`](https://github.com/tensorflow/tensorflow/blob/r1.3/tensorflow/c/c_api.h#L837) provides the required functionality. So, unless this separate API provides some strong benefit (which I don't yet think it does), I'd advise against adding it.